### DISCUSSION TITLE
Hide minimized window snapshot in dock on MacOS

### DISCUSSION THREAD

**Discussion by BlackCoyote** [AvaloniaUI/Avalonia#17142] [Q&A]
When i set the `ShowInDock` property in the `MacOSPlatformOptions` to false to stop my app from showing in dock, it does stop the app from showing an icon in the dock initially, however when the window gets minimized it will still show a little snapshot image of the app in the dock. Is there a way to disable this behaviour?

![image](https://github.com/user-attachments/assets/9631064f-93cb-4ef5-9314-4c7702fa5989)

**Comment by workgroupengineering** [comment]
Did you add `LSUIElement` to `Info.plist`?

**Reply by BlackCoyote** [reply]
When I tried this, it did the same as `ShowInDock` and hid the main icon on the dock, but it did not prevent the miniature window from appearing in the dock when the app's window is minimized.

Perhaps I did it wrong. Is this definitely supposed to prevent the miniature as well?
